packages/kv-asset-handler/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
I'll leave it up to you, but in case we are not planning on fixing those failures any time soon, we might want to consider flagging what you said ☝️ with a comment, just in case someone wakes up one day with the urge of using this config 😅 . Again, abs up to you!

Reply:
I plan on doing it myself soon, filed https://github.com/cloudflare/workers-sdk/issues/6101